src/System.Net.Mime/src/System/Net/Mime/MimeBasePart.cs

```diff
@@ -0,0 +1,289 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Specialized;
+using System.Text;
+using System.Net.Mail;
+
+namespace System.Net.Mime
+{
+    internal class MimeBasePart
+    {
+        internal const string DefaultCharSet = "utf-8";//"iso-8859-1";
```

Comment:
What does the `//"iso-8859-1";` comment mean? Is it dead code that should be removed?